zeroconf/test.py

```diff
@@ -1353,3 +1373,7 @@ def send(out, addr=r._MDNS_ADDR, port=r._MDNS_PORT):
 
     # unregister
     zc.unregister_service(info)
+
+
+if __name__ == '__main__':
```

Comment:
Sorry, but this is also out of scope of this pull request.

Reply:
OK, will remove, will you accept this in another PR?

Reply:
I don't think it'd be useful – even now I don't think we can fully run the test suite without `nosetests` (not every test is in a test method in a subclass of `unittest.TestCase`) and I plan to change it further in the future (less subclassing of `TestCase`).